core/src/avm2/globals/RegExp.as

```diff
@@ -0,0 +1,31 @@
+package {
+    [Ruffle(InstanceAllocator)]
+    [Ruffle(CallHandler)]
+    public dynamic class RegExp {
+        public function RegExp(re:* = undefined, flags:* = undefined) {
+            this.init(re, flags)
+        }
+
+        private native function init(re:*, flags:*):void;
+
+        public native function get dotall():Boolean;
+        public native function get extended():Boolean;
+        public native function get global():Boolean;
+        public native function get ignoreCase():Boolean;
+        public native function get multiline():Boolean;
+        public native function get lastIndex():Number;
+        public native function set lastIndex(value:Number):void;
+        public native function get source():String;
+
+        AS3 native function exec(str:String):Object;
+        AS3 native function test(str:String):Boolean;
+
+        prototype.exec = function(str:String):Object {
```

Comment:
`test` and `exec` should both take an empty string as a parameter by default (in both the `AS3` and the `prototype` methods)